app/components/contact-API.js

```diff
@@ -0,0 +1,34 @@
+
+export const ContactApi = {
+  //making an array to hold Contacts
+  contacts: [
+    {
+      id: 70219577,
+      name: 'Albert Einstein',
+      image_url: "https://images.squarespace-cdn.com/content/v1/62ec2bc76a27db7b37a2b32f/625c7248-8056-4505-a1da-a1058c830d92/albert-einstein-with-blue-hair-large.jpg",
+      email: 'aeinstein@example.com',
+      phone_number: '1555555555'
+
+    },
+    {
+      id:12345678,
+      name: 'Snoopy',
+      image_url: 'https://upload.wikimedia.org/wikipedia/en/5/53/Snoopy_Peanuts.png',
+      email: 'snoopy@example.com',
+      phone_number: '1222222222'
```

Comment:
You are using snake case (phone_number) here which is mostly just used in Python. JS we recommend using camel case (phoneNumber).